src/features_finder.rs

```diff
@@ -87,6 +87,21 @@ pub fn fetch_feature_sets(package: &crate::cargo_metadata::Package) -> Vec<Featu
                 // skip_feature_set matches: do not add it to feature_sets
                 continue 'outer;
             }
+            for feat in feature_set.clone() {
+                for dep in package
+                    .feature_map
+                    .get(&feat.0)
+                    .unwrap_or(&FeatureList::default())
+                    .iter()
+                {
+                    if dep.strip_prefix("dep:").is_some() {
```

Comment:
note that this won't work for old-style implicit dependency features

but we haven't been careful about that in this tool anyway

Reply:
The same mechanism is used in line 22. So I would leave this as is, and an issue could be opened discussing this further. I have no knowledge about the old-style you are referring to, so I personally would not open that issue ;)